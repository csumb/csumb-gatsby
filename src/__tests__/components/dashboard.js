import React from 'react'
import renderer from 'react-test-renderer'
import {
  DashboardEvents,
  DashboardEvent,
  DashboardMessages,
  DashboardMessage,
  DashboardApps,
  DashboardAppsWrapper,
  DashboardApp,
} from 'components/dashboard'

describe('DashboardMessage', () => {
  it('renders messages correctly', () => {
    const tree = renderer
      .create(
        <DashboardMessage
          message={{ title: 'the title', message: 'message' }}
        />
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})

describe('DashboardEvent', () => {
  it('renders messages correctly', () => {
    const tree = renderer
      .create(
        <DashboardEvent
          event={{
            headline: 'the title',
            dashboard: 'message',
            date: 'October 23, 2018',
            link: '#link',
          }}
        />
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
