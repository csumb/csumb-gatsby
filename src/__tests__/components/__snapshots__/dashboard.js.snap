// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DashboardMessage renders messages correctly 1`] = `
<div
  className="css-tb7oa0"
>
  <button
    className="css-2p2q71"
    onClick={[Function]}
  >
    Ã—
    <div
      style={
        Object {
          "border": 0,
          "clip": "rect(0 0 0 0)",
          "height": "1px",
          "margin": "-1px",
          "overflow": "hidden",
          "padding": 0,
          "position": "absolute",
          "width": "1px",
        }
      }
    >
      Archive message
    </div>
  </button>
  <h3>
    the title
  </h3>
  <p>
    message
  </p>
</div>
`;
